httphandler/main.go

```diff
@@ -121,14 +121,14 @@ func initializeSaaSEnv() {
 	}
 
 	backendServices, err := servicediscovery.GetServices(
-		servicediscoveryv1.NewServiceDiscoveryFileV1(serviceDiscoveryFilePath),
+		servicediscoveryv2.NewServiceDiscoveryFileV2(serviceDiscoveryFilePath),
 	)
 	if err != nil {
 		logger.L().Fatal("failed to get backend services", helpers.Error(err))
 		return
 	}
 
-	if ksCloud, err := v1.NewKSCloudAPI(backendServices.GetReportReceiverHttpUrl(), backendServices.GetApiServerUrl(), config.GetAccount(), config.GetAccessKey()); err != nil {
+	if ksCloud, err := v1.NewKSCloudAPI(backendServices.GetApiServerUrl(), backendServices.GetReportReceiverHttpUrl(), config.GetAccount(), config.GetAccessKey()); err != nil {
```

Comment:
this is a bug where the report URL and API server URLs were sent in different order